front/app/containers/App/CommunityMonitorModal/index.tsx

```diff
@@ -128,7 +129,9 @@ const CommunityMonitorModal = ({
   }, [shouldShowModal]);
 
   const onClose = () => {
-    set('community_monitor_modal_seen', 'true');
+    set('community_monitor_modal_seen', 'true', {
+      expires: 90, // Cookie expires by default after 90 days
+    });
```

Comment:
Can we make this expire after 3 months? Fear this may conflict with the backend, where logic to reject responses is all locked to 3 months